build/source/engine/groundwatr.f90

```diff
@@ -238,6 +251,9 @@ USE soil_utils_module,only:hydCond_psi         ! compute hydraulic conductivity
    ! compute baseflow flux
    call computeBaseflow(&
                         ! input: control and state variables
+                        nSnow,                     & ! intent(in): number of snow layers
+                        nSoil,                     & ! intent(in): number of soil layers
+                        nLayers,                   & ! intent(in): total number of layers
                         .false.,                   & ! intent(in): .true. if derivatives are desired
```

Comment:
Yes. This is an optional second call to `computeBaseflow` -- the purpose is to compute the numerical derivatives, so we do not need to compute analytical derivatives also.